chatbot_supervised.py

```
@@ -0,0 +1,504 @@
+from typing import Annotated, Sequence, List, Literal ,Optional ,TypedDict
```

Ruff (0.11.9)

`typing.Sequence` imported but unused

Remove unused import: `typing.Sequence`

(F401)